pkg/transforms/event_test.go

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright (c) 2022 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package transforms
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/requests"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEventWrapper_Wrap(t *testing.T) {
+	obj := struct {
+		myInt int
+		myStr string
+	}{
+		myInt: 3,
+		myStr: "hello world",
+	}
+	tests := []struct {
+		Name          string
+		ProfileName   string
+		DeviceName    string
+		ResourceName  string
+		ValueType     string
+		MediaType     string
+		Data          interface{}
+		ExpectedError interface{}
+	}{
+		{"Successful Binary Reading", "MyProfile", "MyDevice", "BinaryEvent", common.ValueTypeBinary, "stuff", true, ""},
+		{"Successful Object Reading", "MyProfile", "MyDevice", "ObjectEvent", common.ValueTypeObject, "", obj, ""},
+		{"Successful Simple Reading", "MyProfile", "MyDevice", "ObjectEvent", common.ValueTypeObject, "", "hello there", ""},
```

Comment:
Shouldn't this be String type?
```suggestion
		{"Successful Simple Reading", "MyProfile", "MyDevice", "ObjectEvent", common.ValueTypeString, "", "hello there", ""},
```